webapp/channels/src/components/post/post_options.tsx

```diff
@@ -59,27 +59,35 @@ type Props = {
 };
 
 const PostOptions = (props: Props): JSX.Element => {
-    const dotMenuRef = useRef<HTMLDivElement>(null);
-
     const [showEmojiPicker, setShowEmojiPicker] = useState(false);
     const [showDotMenu, setShowDotMenu] = useState(false);
     const [showActionsMenu, setShowActionsMenu] = useState(false);
 
-    const toggleEmojiPicker = useCallback(() => {
-        setShowEmojiPicker(!showEmojiPicker);
-        props.handleDropdownOpened!(!showEmojiPicker);
-    }, [props.handleDropdownOpened, showEmojiPicker]);
+    const toggleEmojiPicker = useCallback((show: boolean) => {
+        setShowEmojiPicker(show);
+        props.handleDropdownOpened!(show);
+    }, [props.handleDropdownOpened]);
 
+    const lastEmittedFrom = useRef(props.shortcutReactToLastPostEmittedFrom);
```

Comment:
For some reason, this `useEffect` started to trigger twice before `shortcutReactToLastPostEmittedFrom` is cleared which caused the hotkey to immediately close the emoji picker after opening it, so we now make sure that it only runs when that one prop changes